src/components/Result.svelte

```diff
@@ -33,9 +33,13 @@
     <span class="favicon-placeholder" />
   {/if}
 
-  <div class="info" title={result.title}>
-    <h4 class="title">{result.title}</h4>
+  <div class="info" title={result.title}> 
+    <h4 class="title">{result.title}</h4> 
+    {#if result.title}
     <p class="url">{result.url}</p>
+    {:else}
+    <p class="url-2">{result.url}</p>
+    {/if}
```

Comment:
We need not duplicate the URL element and the CSS classes here.
Using Svelte, we can dynamically apply an inline-style to set `margin-top` to 0 when the title is not there.
Example:
```svelte
<p class="url" style={!result.text ? 'margin-top: 0' : ''}>
```

Reply:
Made the changes. Thanks.